xlml/utils/xpk.py

```diff
@@ -79,17 +73,20 @@ def run_workload(
       ),
   )
 
-  return KubernetesPodOperator(
-      task_id=task_id,
-      name=benchmark_id,
-      cmds=["/bin/bash", "-c"],
-      arguments=[";".join(cmds)],
+  subprocess.run(["bash", "-c", ";".join(cmds)], check=True)
+
+
+# To support local execution using `scripts/local-airflow.sh`, run using
+# task.docker in local environments. Otherwise, task.kubernetes should be used.
+if "XLMLTEST_LOCAL_AIRFLOW" in os.environ:
```

Comment:
I have a design question: do we want to have one flag for local airflow that enables a bunch of features (this), or individual flags for each relevant feature (`XLMLTEST_SSH_EXTERNAL_IPS`)?

cc @RissyRan

Reply:
That's a good point, for this change I only need to know if we're local or not. If that's generally the case, I would be in favor of a single env var to denote execution environment (standalone or Composer).

Reply:
One flag sounds better to me.

Reply:
We'll consolidate in a separate change.

Reply:
Thanks Jon!